lib/Element.js

```diff
@@ -88,6 +88,19 @@ Element.prototype = Object.create(ContainerNode.prototype, {
       }
     }
   },
+  innerText: {
+    get: function() {
+      var strings = [];
+      recursiveGetText(this, strings);
+      // Strip and collapse whitespace
+      return strings.join('').replace(/[ \t\n\f\r]+/g, ' ').trim();
+    },
+    set: function(newtext) {
+      if (newtext !== null && newtext !== undefined && newtext !== '') {
```

Comment:
Should we also clear an existing content here before appending a new text node (similar to the `textContent` above)? If not, it'd be helpful to have a quick comment in the code to explain why (for future context).

Reply:
We should. good catch